### ISSUE TITLE
About package release and tags

### ISSUE THREAD

**Issue by Hayao0819** [nsz32/docklike-plugin#28]
Currently the AUR has only xfce4-docklike-plugin-git which builds the latest version. It is a version during development and may be unstable.
I would like to create tags as a stable stage in this project and release them regularly.
I think this will allow users to separate stable versions from unstable-git.

**Comment by nsz32** [comment]
Yep, it's time to make such thing.
Did you know if the AUR package can ask if the user want stable or unstable code ?

**Comment by Hayao0819** [comment]
I'm sorry I don't understand English well, so I don't know if I can interpret it correctly.
Does it mean that users can choose between stable and unstable versions?

**Comment by nsz32** [comment]
Yes, I propose to keep one AUR package that let the user choose between stable and unstable version. If this is possible ...

**Comment by nsz32** [comment]
Also the stable tag risk to block some new translations in the future.
I don't plan to implement many new features for the next month, so, we can restart this discussion after some translations are integrated.

**Comment by Hayao0819** [comment]
understood. After some translations are integrated, 
I hope the release is done (the tags are created) and a stable package is created.

**Comment by davekeogh** [comment]
Hey @nsz32 , 

I thought I'd bump this.  What do you think still needs doing before making a release? I can take care of some of the less complex coding tasks, but some of it is beyond me.

**Comment by davekeogh** [comment]
Hi. I've never done a release of any kind of software and it seems nsz32 is not available to do any work towards this ATM. I could generate some tarballs on my github page. The only significant, non-feature request issue is #90, but I don't have the c++ expertise to resolve it.

**Comment by philipzae** [comment]
@SwampRabbit can you assist with this effort, as I see that dolphinoracle is also using it.

**Comment by davekeogh** [comment]
I've created some packages over on my github page. Give these a try:
https://github.com/davekeogh/xfce4-docklike-plugin/releases/tag/v0.2.0

**Comment by Hayao0819** [comment]
I have created a package with @davekeogh 's tag.
They are working fine, at least in my environment.

https://aur.archlinux.org/packages/xfce4-docklike-plugin/

**Comment by SwampRabbit** [comment]
@philipzae apologies, was away from dev/packaging work for awhile due to real life.

When we package something we stick to the developer's release number and append distro specifics.
Technically, any distro can package something with or without a release number, in cases where there isn't, then for example we can use the git version number.

So I can't really help set the release number or anything.  But I can say that what @davekeogh is doing is perfectly fine.
Additionally, the 0.2.0 version builds and seems to work without issue on MX-19 / Debian 10.
